### DISCUSSION TITLE
Calculating RMSF for a trajectory of a tetrameric protein.

### DISCUSSION THREAD

**Discussion by FezokaTurbo** [MDAnalysis/mdanalysis#4616] [Q&A] [answered]
Greetings! 
First of all, it has been a great experience to learn Python with things I actually do IRL, working in proteomics. MDAnalysis is helping me a lot to understand the language.
Very well, my issue: As the title says, I'm having issues for calculating and plotting the RMSF for the trajectory of my protein. I believe it's because, for each monomer, the residue number restarts. For example: The M residue for the first monomer is 0, while it is also 0 for the other 3. Because of that, my plot image with matplotlib appears to have 4 results for a single residue number.

The question is: 
Is it possible to separate each monomer using MDAnalysis, as our MD was made using all four monomers together?
Any suggestion on how I should proceed with this?

**Comment by RMeli** [comment, marked as answer]
Hello! Have you tried using the `segid` and `fragment` selection? Otherwise it should be possible to re-number the residues IDs.

**Reply by orbeckst** [reply]
I might also help if you posted your code.
